R/topics.R

```diff
@@ -167,3 +167,69 @@ topic_must <- function(message, topic) {
     x = paste0("Not ", encodeString(topic, quote = "'"))
   ))
 }
+
+content_info <- function(content_entry, index, pkg, section) {
+
+  if (!grepl(".*::.*", content_entry)) {
+    topics <- pkg$topics[select_topics(content_entry, pkg$topics),]
+    tibble::tibble(
+      path = topics$file_out,
+      aliases = purrr::map2(topics$funs, topics$name, ~ if (length(.x) > 0) .x else .y),
+      name = list(topics$name),
+      title = topics$title,
+      icon = find_icons(topics$alias, path(pkg$src_path, "icons"))
+    )
+  } else { # topic from another package
+    names <- strsplit(content_entry, "::")[[1]]
+    pkg_name <- names[1]
+    topic <- names[2]
+    check_package_presence(pkg_name)
+
+    rd_href <- find_rd_href(sub("\\(\\)$", "", topic), pkg_name)
```

Comment:
the user would have to add the `()` to the configuration file, pkgdown ignores them for finding the rd.